pgml-dashboard/src/components/headings/gray/mod.rs

```diff
@@ -0,0 +1,18 @@
+use pgml_components::component;
+use sailfish::TemplateOnce;
+
+#[derive(TemplateOnce, Default)]
+#[template(path = "headings/gray/template.html")]
+pub struct Gray {
+    value: String,
+}
+
+impl Gray {
```

Comment:
If we want to go down this road, maybe one heading that takes a color, or has a setter function for the desired color, rather than a new component for every header.

Reply:
Yeah I agree, these are too small to be individual components.